### PR TITLE
Changed style-related methods

### PR DESCRIPTION
**Pull request by jamesellisnyc**
Changed `Assets.php`, providing more flexibility around getting a manifest's styles. Previously one could only get a string of `<link>` tags. The new responsive _cardio_ site structure requires that we get an array of the style sheets so they can be rendered programmatically.

You can see this WIP here:
https://github.com/mdx-dev/vitals/blob/cardio/style_and_wrappers/app/twig_views/elements/head_logic/head_logic.twig#L16

In summary, what's changed:
- Updated `style` method
- Introduced `style_list` method
- Introduced `gen_css_files` method used by both
- Updated documentation throughout

Note: The changes have been discussed and reviewed by @blainesch.

### CODE REVIEW

**Comment by bayleedev** [comment]
:+1:

**Comment by elijahsummers** [comment]
:+1:

**Comment by jamesellisnyc** [comment]
@blainesch & @elijahsummers — Thanks for the feedback. Do I have the all clear to merge or is there a [merge master](http://youtu.be/N9L7UUp0FxY?t=13s) responsible for merging?

**Comment by elijahsummers** [comment]
@jamesellisnyc, go for it. We don't have any special person responsible for merging. In general, once a PR gets reviewed, it's up to the original dev to decide when it gets merged.
